releaseNotes/PythonFeatureManagement.md

```diff
@@ -0,0 +1,19 @@
+# featuremanagement
+
+[Source code][source_code] | [Samples][samples]
+
+## 1.0.0b1 - May 22, 2024
+
+Initial Beta Release of the Feature Management for Python
+
+### New Features
+
+* Loading of feature flags from a file, see [feature management schema].
+* Loading of feature flags from Azure App Configuration provider.
```

Comment:
I guess the first item is wrong, as we support json, which has been loaded from a file into a dictionary.

As for the second item, I thought it would be good just as a callout that it is supported, though it will need 1.2.0 or later.

Reply:
The FM library doesn't read files, does it?

>  it will need 1.2.0 or later

This is something worth calling out. How about something like this?

Initial release of the feature management support for Python. Note that, version 1.2.0 or later of xxx (provider package name) is required for loading feature flags from Azure App Configuration.